package.json

```diff
@@ -4,7 +4,7 @@
   "description": "absolution accepts HTML and a base URL, and returns HTML with absolute URLs. Great for generating valid RSS feeds.",
   "main": "index.js",
   "scripts": {
-    "test": "mocha test/test.js"
+    "test": "node_modules/mocha/bin/mocha"
```

Comment:
Actually this change is not necessary, when running from an npm task the node_modules bin folder comes first. The devDependency change is needed though.

Reply:
We can also simplify the command all the way to just "mocha". Standard behavior of mocha is to run everything in "test".

Reply:
One last thing... the dev dep on mocha is out of date (yes it's been a while).